gui.py

```
@@ -1,494 +1,49 @@
-        self.vars = []
-        self.make_checkbuttons(items)
+    logo_image = ft.Image(
+        src=str(ASSETS_DIR / "logo.png"),
+        width=200,
+        height=200,
+        fit="contain",
+    )
+    try: 
+         # again this only works for compiled so we dont wanna explode it if it fails in regular python
+        page.window.icon = str(ASSETS_DIR / "icon.ico")
+    except:
```

Ruff (0.15.10)

[error] Do not use bare `except`

(E722)